src/resources/Witchess.ts

```diff
@@ -1,8 +1,36 @@
+import { myHash, runChoice, runCombat, toInt, visitUrl } from "kolmafia";
 import { haveInCampground } from "../lib";
+import { get } from "../property";
 import { $item } from "../template-string";
 
-export const item = $item`Witchess Set`;
```

Comment:
I kinda liked being able to do `witchess.item`, but I can see why we might want to remove this

Reply:
fixed